### ISSUE TITLE
Unable to create install disk with Big Sur releases

### ISSUE THREAD

**Issue by varszegimarcell** [corpnewt/gibMacOS#194]
Hi!

I've noticed some odd behavior when I try to create a Big Sur installer using this script. When I try to run BuildmacOSInstallApp.command, and I drop the downloaded big sur folder, I get this result:
![Képernyőfotó 2020-11-16 - 23 51 02](https://user-images.githubusercontent.com/47447944/99318027-40de8700-2867-11eb-92a4-d164750da90a.png)
![Képernyőfotó 2020-11-16 - 23 51 18](https://user-images.githubusercontent.com/47447944/99318031-42a84a80-2867-11eb-9d6a-36f4747f009e.png)

Now, as I've noticed, these files really doesn't existing in the downloaded folder, although the download was successful, and error free.

![Képernyőfotó 2020-11-16 - 23 57 08](https://user-images.githubusercontent.com/47447944/99318143-71262580-2867-11eb-8413-30b72e31f62c.png)

Can the installer media structure changed that much with Big Sur? Are there any known fixes for this? Or will you have to modify the installer script?

Thanks in advance for your help.

**Comment by egtyzh** [comment]
https://dortania.github.io/OpenCore-Install-Guide/extras/big-sur/#grabbing-the-installer

**Comment by varszegimarcell** [comment]
> 
> 
> https://dortania.github.io/OpenCore-Install-Guide/extras/big-sur/#grabbing-the-installer

Yes, I knew about it, but the whole point of using gibMacOS script was that you were able to make install disks on Windows and Linux. Now with Big Sur you cannot, that's why I opened this issue. With the new version, gibMacOS cannot do anything, except downloading it in a format that is just useful for Mac machines.  

BTW, as you can see, I'm already on Big Sur, with an installer grabbed from Apple.

**Comment by wangxh86** [comment]
> > https://dortania.github.io/OpenCore-Install-Guide/extras/big-sur/#grabbing-the-installer
> 
> Yes, I knew about it, but the whole point of using gibMacOS script was that you were able to make install disks on Windows and Linux. Now with Big Sur you cannot, that's why I opened this issue. With the new version, gibMacOS cannot do anything, except downloading it in a format that is just useful for Mac machines.
> 
> BTW, as you can see, I'm already on Big Sur, with an installer grabbed from Apple.

How did you solve it? I have the same problem now, if you can see it, can you give me a solution? Thank you in advance

**Comment by varszegimarcell** [comment]
> > > https://dortania.github.io/OpenCore-Install-Guide/extras/big-sur/#grabbing-the-installer
> > 
> > 
> > Yes, I knew about it, but the whole point of using gibMacOS script was that you were able to make install disks on Windows and Linux. Now with Big Sur you cannot, that's why I opened this issue. With the new version, gibMacOS cannot do anything, except downloading it in a format that is just useful for Mac machines.
> > BTW, as you can see, I'm already on Big Sur, with an installer grabbed from Apple.
> 
> How did you solve it? I have the same problem now, if you can see it, can you give me a solution? Thank you in advance

I installed catalina first, than I made an USB installer via the App store, the official way. Since it does not create the EFI partition, I used an another USB disk with gibMacOS buildInstall.command. I used the option make bootloader only, and with opencore. (So it looked like in my case: 1 B O) I used the two USB disks to make the installation happen. 

Now, you have the option to update from Catalina Sytem preferences, but since I’m using a Haswell machine, the update fails, and boots straight back to Catalina as nothinq would happened, so my only option is here to perform a clean install. (BTW, Haswell Macbook Pros brick from Big Sur, so yolo, something is not alright when you update on Haswell. At least it cant brick a hankintosh xD)

**Comment by wangxh86** [comment]
Great, thank you very much.
